lib/ui/painting.dart

```diff
@@ -1558,21 +1586,41 @@ class Path extends NativeFieldWrapperClass2 {
 
   /// Adds a new subpath that consists of the given path offset by the given
   /// offset.
-  void addPath(Path path, Offset offset) {
+  /// 
+  /// If matrix4 is specified, the path will be transformed by this matrix
+  /// (after the matrix is translated by the given offset).
```

Comment:
Mention here and in the docs for any other functions that takes a Float64List matrix that the matrix argument is a 4x4 matrix, and say if it's row-major or column-major, etc.